Mathlib/CategoryTheory/Limits/Shapes/Biproducts.lean

```diff
@@ -258,7 +258,7 @@ structure IsBilimit {F : J → C} (B : Bicone F) where
 
 attribute [inherit_doc IsBilimit] IsBilimit.isLimit IsBilimit.isColimit
 
--- Porting note: simp can prove this, linter doesn't notice it is removed
+-- Porting note (#10618): simp can prove this, linter doesn't notice it is removed
```

Comment:
And this one!

Reply:
Now this question can be found [here](https://github.com/leanprover-community/mathlib4/issues/10618#issuecomment-1955023023).